test/parallel_api/ranges/reduce_by_key_perf_host.pass.cpp

```diff
@@ -0,0 +1,112 @@
+// -*- C++ -*-
```

Comment:
The host test also should be move together with host version as I suggested above.